packages/cli/src/lib/AndroidSdkToolsInstaller.ts

```diff
@@ -15,28 +15,36 @@
  */
 
 import * as path from 'path';
-import * as util from '../util';
+import {util} from '@bubblewrap/core';
+import {Prompt} from './Prompt';
+import {Presets, Bar} from 'cli-progress';
+import {green} from 'colors';
+import {enUS as messages} from './strings';
 
 const SDK_VERSION = '6609375';
 const DOWNLOAD_SDK_ROOT = 'https://dl.google.com/android/repository/';
 const WINDOWS_URL = `commandlinetools-win-${SDK_VERSION}_latest.zip`;
 const MAC_URL = `commandlinetools-mac-${SDK_VERSION}_latest.zip`;
 const LINUX_URL = `commandlinetools-linux-${SDK_VERSION}_latest.zip`;
+const ANDROID_SDK_SIZE = 86532338;
```

Comment:
I think this is OK, but I wonder if there's anything we can do to make sure it's always updated when SDK_VERSION is updated.

Reply:
My first attempt was to use the `content-length` header, so I wouldn't have to hard-code the size, but it's not always there. As a matter of fact, this PR is missing the fact that Windows, MacOS and Linux versions likely have different sizes.

Reply:
I'm only using the fixed size for the JDK sources. Using Content-Length for JDK Bin and Android SDK.